src/util/expr.h

```diff
@@ -13,6 +13,7 @@ Author: Daniel Kroening, kroening@kroening.com
 #define OPERANDS_IN_GETSUB
 
 #include "type.h"
+#include <memory>
```

Comment:
std::reference_wrapper requires that. Will try 2) or replace reference_wrapper with pointers.